xrpl/account/__init__.py

```diff
@@ -1,10 +1,21 @@
 """Public interface for XRPL account sugar methods."""
 
 from xrpl.account.main import (
+    does_account_exist,
     get_account_info,
     get_balance,
-    get_fee,
     get_next_valid_seq_number,
 )
+from xrpl.account.transaction_history import (
+    get_account_payment_transactions,
+    get_account_transactions,
+)
 
-__all__ = ["get_fee", "get_next_valid_seq_number", "get_balance", "get_account_info"]
+__all__ = [
+    "get_next_valid_seq_number",
+    "get_balance",
+    "get_account_info",
+    "get_account_payment_transactions",
+    "get_account_transactions",
+    "does_account_exist",
```

Comment:
i can't think of a better name, but something about this bothers me. @hammertoe - do you have any thoughts on naming functions in python that return bools?

Reply:
Yeah I agree, bool functions typically start with `is`, but I couldn't figure out the right wording for that. For example, `is_account_existing` and `is_account_in_existence` feel a bit awkward.